docs/Player.html

```diff
@@ -278,7 +278,7 @@ <h4 class="name" id="bytesProcessed"><span class="type-signature"></span>bytesPr
     
     <dt class="tag-source">Source:</dt>
     <dd class="tag-source"><ul class="dummy"><li>
-        <a href="player.js.html">player.js</a>, <a href="player.js.html#line443">line 443</a>
+        <a href="player.js.html">player.js</a>, <a href="player.js.html#line447">line 447</a>
```

Comment:
As a suggestion, it might be worth removing the docs directory from check in to avoid the noise in PRs (and maybe autodeployed from CI).

Reply:
Agreed!